airflow/dags/parse_elavon/elavon_to_gcs_jsonl.py

```diff
@@ -26,19 +26,22 @@ def fetch_and_clean_from_gcs():
     all_rows = pd.DataFrame()
 
     # List raw files available from GCS
-    gfs = gcsfs.GCSFileSystem(
-        project="cal-itp-data-infra",
-        token=BIGQUERY_KEYFILE_LOCATION,
-    )
-    file_list = gfs.ls("test-calitp-elavon-raw/", detail=False)
+    file_and_dir_list = fs.ls("test-calitp-elavon-raw/", detail=False)
```

Comment:
Should this be an env var?

Reply:
Resolved in [8563ed3](https://github.com/cal-itp/data-infra/pull/2376/commits/8563ed3bf26e1d81ea93c4c64c796c7fe263cecd)